src/Helper/Twig/TwigLoader.php

```diff
@@ -33,33 +33,23 @@ public function __construct(ClientRequest $client, array $config)
     /**
      * {@inheritdoc}
      */
-    public function getSourceContext($name)
-    {
-        return new \Twig_Source($this->getTemplateCode($name), $name);
-    }
-
-    /**
-     * {@inheritdoc}
-     *
-     * @deprecated used for php < 7
-     */
-    public function getSource($name)
```

Comment:
Delete?

Reply:
Yes was only needed in version before php7